### DISCUSSION TITLE
Use only every second waste pickup?

### DISCUSSION THREAD

**Discussion by wildpash** [mampfes/hacs_waste_collection_schedule#1620] [General]
In our district, out of 24 pickup appointments, only 12 are free. Is it possible to edit the source so that, for example, only every second pickup date is processed or imported? (ICS sources: art_trier_de)

**Comment by 5ila5** [comment]
There is no easy way to archive this, if all events have the same name.  If they have different names you can use the [customize: show: False arguemnt](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md#attributes-for-customize).

To be able to only return every second collection we would need to add an additional parameter to the ics source / to our Service class.
